lib/xnu/README.md

```diff
@@ -8,9 +8,9 @@ Experimental kernel driver for boosting Frida's capabilities on macOS:
 
 ## Installation
 
-1. Open `FridaKernelAgent.xcodeproj` and build it with Xcode
-2. Review `reload.sh`, adjusting it to match your system
-3. Run `reload.sh` to install and load the driver
+Run `reload.sh` to install and load the driver
+If it fails to load, set `CODE_SIGN_IDENTITY` enviroment variable to
+identity valid for Kext signing, or disable SIP
```

Comment:
Sweet! How about reflowing to 72 cols, with full stops between sentences? (Which in hindsight I should have done in my list.)